### ISSUE TITLE
How do I get the LABEL data

### ISSUE THREAD

**Issue by celia916** [vlkniaz/ThermalGAN#6]
I saw that the code only combines the LABEL and the original color image and input to the pix2pix model, but I didn't know how to get the LABEL. The paper said that it was acquired by BicycleGAN. How to get the LABEL by code?

**Comment by silence401** [comment]
Do you know how to get  it  now?

**Comment by wuancong** [comment]
The project page seems not available now. Can the dataset be downloaded? Thank you!

**Comment by heitorrapela** [comment]
I am also having the same problem generating the thermal images from my own dataset using the pre-trained weights. I will try one more time, but I don't think that I will succeed :)

**Comment by nauyan** [comment]
@heitorrapela @silence401 @celia916 you able to use the model on your own dataset?

**Comment by heitorrapela** [comment]
No, and you?

**Comment by nauyan** [comment]
@heitorrapela Nope
